test-suite/Math/NumberTheory/Powers/CubesTests.hs

```diff
@@ -0,0 +1,75 @@
+-- |
+-- Module:      Math.NumberTheory.Powers.CubesTests
+-- Copyright:   (c) 2016 Andrew Lelechenko
+-- Licence:     MIT
+-- Maintainer:  Andrew Lelechenko <andrew.lelechenko@gmail.com>
+-- Stability:   Provisional
+--
+-- Tests for Math.NumberTheory.Powers.Cubes
+--
+
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
+
+module Math.NumberTheory.Powers.CubesTests
+  ( testSuite
+  ) where
+
+import Test.Tasty
+import Test.SmallCheck.Series
+
+import Data.Functor.Identity
```

Comment:
Yeah, good idea to replace `Identity` with newtype. 

`Positive` and `NonNegative` are from `Test.SmallCheck.Series`. That is why we need to write orphan instances to quickcheck them.